endpoints.py

```diff
@@ -259,6 +289,35 @@ async def create_plugin_zip(repo_path: str, plugin_name: str):
 
         return zip_filename
 
+    
+    @staticmethod
+    async def download_releses_plugin_zip(plugin_name: str, url_zip: str, version_origin: int):
+        # Define a cache directory
+        cache_dir = "zip_cache"
+        if not os.path.exists(cache_dir):
+            os.makedirs(cache_dir)
+        
+        name_plugin = plugin_name + ".zip"
+        os_path_plugin = os.path.join(cache_dir, name_plugin)
+        
+        check = check_version_zip(plugin_name,version_origin)
```

Comment:
this function is both returning a boolean and updating `versions.json`, which is confusing